docs/EXAMPLES.md

```diff
@@ -66,11 +66,33 @@ describe("foo", () => {
 });
 ```
 
+## waitForAlertToBePresent
+
+| 1 optional argument                     |
+|-----------------------------------------|
+| `timeoutInMilliseconds` (default: 5000) |
+
+```js
+const protractorHelper = require("protractor-helper");
+
+describe("foo", () => {
+  it("bar", () => {
+    browser.get("https://example.com");
+
+    const alertElement = element(by.className("el-alert"));
```

Comment:
Hey, @pedrohyvo, an alert can't be located as an HTML element.

Here we should have something like this.

```js
const buttonThatTriggersAnAlert = element(by.css('button.some-button'));

helper.click(buttonThatTriggersAnAlert);
```

Then what's already on line 84.

:wink:

Reply:
Done.